### PR TITLE
chore: fix development issues on Windows

### LINKED ISSUE #14014: `integration::run::issue9750` gets stuck forever
When I run `cargo test` on my machine, I get a message about `integration::run::issue9750` timing out, and `cargo test` never finishes. When I run that test specifically, it also never finishes running, but if you print its stdout (with `-- --nocapture`), you see that the test in fact fails:

```
OUTPUT Enter \'yy\':\r\n
INPUT yy\n
ECHO: yy\r\n
OUTPUT ⚠️  ️Deno requests env access. Run again with --allow-env to bypass this prompt.\r\n   Allow? [y/n (y = yes allow, n = no deny)]  
INPUT n\n
ECHO: n\r\n
OUTPUT ⚠️  ️Deno requests env access to \"SECRET\". Run again with --allow-env to bypass this prompt.\r\n   Allow? [y/n (y = yes allow, n = no deny)]  
thread 'integration::run::issue9750' panicked at 'assertion failed: `(left == right)`

Diff < left / right > :
<⚠️  ️Deno requests env access to "SECRET". Run again with --allow-env to bypass this prompt.
>⚠️  ️Deno requests env access to "SECRET". Run again with --allow-run to bypass this prompt.
    Allow? [y/n (y = yes allow, n = no deny)]

', test_util/src/lib.rs:1910:11
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

The actual permissions prompt correctly asks for `--allow-env`, but [the test expects it to ask for `--allow-run`](https://github.com/denoland/deno/blob/main/cli/tests/integration/run_tests.rs#L2337).

I don't know why the test keeps running, though, or why this hasn't been observed in CI.

### PR DESCRIPTION
**Pull request by dsherret**
Fixes some local development issues that have creeped up over the past few weeks on Windows.

### Please merge on review

Closes #14014

### CODE REVIEW

File: cli/tests/integration/run_tests.rs [hunk 1]
```diff
@@ -2334,7 +2334,7 @@ fn issue9750() {
       Input("yy\n"),
       Output("⚠️  ️Deno requests env access. Run again with --allow-env to bypass this prompt.\r\n   Allow? [y/n (y = yes allow, n = no deny)]"),
       Input("n\n"),
-      Output("⚠️  ️Deno requests env access to \"SECRET\". Run again with --allow-run to bypass this prompt.\r\n   Allow? [y/n (y = yes allow, n = no deny)]"),
+      Output("⚠️  ️Deno requests env access to \"SECRET\". Run again with --allow-env to bypass this prompt.\r\n   Allow? [y/n (y = yes allow, n = no deny)]"),
```
Comment (dsherret) [inline cli/tests/integration/run_tests.rs:2337]: Ummm... how is this passing on linux & mac?

Comment (dsherret) [reply cli/tests/integration/run_tests.rs:2337]: This fails for me on linux and then hangs forever.

File: .cargo/config [hunk 2]
```diff
@@ -1,5 +1,14 @@
 [target.x86_64-pc-windows-msvc]
 rustflags = ["-C", "target-feature=+crt-static"]
 
+[target.'cfg(all(windows, debug_assertions))']
+rustflags = [
+  "-C",
+  "target-feature=+crt-static",
+  "-C",
+  # double the stack size to prevent swc overflowing the stack in debug
+  "link-arg=/STACK:2097152",
```
Comment (bartlomieju) [inline .cargo/config:10]: Again...? 😬

Comment (dsherret) [reply .cargo/config:10]: That happened in the past? FWIW swc’s tests on its repo requires increasing the stack size.

File: cli/tests/integration/run_tests.rs (hunk 1 above)
Comment (kitsonk) [reply cli/tests/integration/run_tests.rs:2337]: See #14014, couldn't figure out why CI wasn't failing but was hanging locally, seems you figured it out.

File: .cargo/config (hunk 2 above)
Comment (bartlomieju) [reply .cargo/config:10]: Yeah, I believe we once increased stack size on Windows, but then there were some improvements that rendered it unneeded

Comment (dsherret) [reply .cargo/config:10]: I think probably something must have increased on our side that just tipped it over. It never used to stack overflow on Windows with this swc version.

**Review by bartlomieju — APPROVED** [review]
LGTM

**Pull request merged**